.teamcity/Gradle_Check/configurations/PerformanceTest.kt

```diff
@@ -48,10 +51,10 @@ class PerformanceTest(model: CIBuildModel, type: PerformanceTestType, stage: Sta
         checkCleanM2()
         if (type.hasRerunner) {
             val rerunnerParameters = listOf(
-                "-PteamCityBuildId=%teamcity.build.id%",
-                "-PonlyPreviousFailedTestClasses=true",
-                "-PgithubToken=%github.ci.oauth.token%")
-            runner("GRADLE_RERUNNER", "tagBuild distributed${type.taskId}s", rerunnerParameters.joinToString(" "))
+                    "-PteamCityBuildId=%teamcity.build.id%",
+                    "-PonlyPreviousFailedTestClasses=true",
+                    "-PgithubToken=%github.ci.oauth.token%")
+            runner("GRADLE_RERUNNER", "tagBuild distributed${type.taskId}s", rerunnerParameters.joinToString(" "), ExecutionMode.ALWAYS)
```

Comment:
Why does `ALWAYS` help here? Will we actually generate the report when the task is up-to-date or from cache in the second run?
I think we should probably use `RUN_ON_FAILURE` as we do for the normal test re-runner. We also need to add a build scan tag to the re-run step, right?

Reply:
Yeah... `RUN_ON_FAILURE` is better. And will add the tag for rerun.